### PR TITLE
Remove environment field from api query and add tests

### PR DESCRIPTION
**Pull request by sjanota**
**Description**

Changes proposed in this pull request:

- Remove deprecated field environments in `apis` query
- Add test for this query

**Related issue(s)**

Issue #2163

### CODE REVIEW

**Comment by sjanota** [comment]
retest

File: tests/ui-api-layer-acceptance-tests/internal/domain/apicontroller/apis_test.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+			},
+		},
+	}
+}
+
+func fixAPIQuery() *graphql.Request {
+	query := `query ($namespace: String!) {
+				apis(namespace: $namespace) {
+					name
+    				hostname
```
Comment (kubadz) [inline tests/ui-api-layer-acceptance-tests/internal/domain/apicontroller/apis_test.go]: Indentation is broken here

**Pull request merged**